playwright/assertions/expect-url.ts

```diff
@@ -0,0 +1,23 @@
+import { expect, test } from '@playwright/test';
+
+// types
+import type { Page } from '@playwright/test';
+
+/**
+ * Asserts that the current URL of the page matches the expected URL.
+ * @function expectUrl
+ * @param {Page} page - The Playwright page object.
+ * @param {url} url - A string representing the expected URL.
+ * @returns {Promise<void>} - A promise that resolves when the assertion is complete.
+ */
+
+export const expectUrl = async (
+    page: Page,
+    url: string
+): Promise<void> => {
+    await test.step('Test the correct url is navigated to', async () => {
+        const urlRegex = new RegExp(url.replaceAll('**', '\\d+'));
```

Comment:
I realize this is a direct copy from the work in CCM... but, looking at it now with an eye for reusability: Should this perhaps replace `**` with `.+`?

Reply:
I've made this change.